addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1455,6 +1490,118 @@ export class ListRenderer extends Component {
+            if(propertiesColumnIndex >= 0) {
+                const baseColumnObject = columns[propertiesColumnIndex];
+                const propertyColumns = this.getPropertyFieldsFromPropertiesField(baseColumnObject);
+                let property_column_id = 1;
+                for(const column of propertyColumns){
+                    column.id = `${column.id}_property_${property_column_id}`;
+                    property_column_id++;
+                }
+                columns.splice(propertiesColumnIndex, 1, ...propertyColumns);
+            }
+        }
+        return columns;
+    }
+
+    /**
+     * Replace 'properties' field with a field for each property value
+     * preserving most of the attributes of the original field.
+     * @param {Object} fields in the form of fieldInfo (e.g. this.props.list.fields)
+     * @returns input or altered copy of the input
+     */
+    propertiesProcessFields(fields) {
+        const propertiesFieldNames = this.getPropertiesFieldNames();
+        if(!propertiesFieldNames) {
+            return fields;
+        }
+        fields = {...fields};
+        for(const propertiesFieldName of propertiesFieldNames){
+            const properties = fields[propertiesFieldName];
+            if(properties) {
+                for(const property of this.getPropertyFieldsFromPropertiesField(properties)){
+                    fields[this.getPropertyFieldName(property.name, propertiesFieldName)] = property;
+                }
+                delete fields[propertiesFieldName];
+            }
+        }
+        return fields;
+    }
+
+    /**
+     * Turns the data value of the Properties fields into individual data values
+     * @param {Object} records
+     * @returns the same records Object with additional data values
+     */
+    propertiesProcessRecords(records) {
+        const propertiesFieldNames = this.getPropertiesFieldNames();
+        if(!propertiesFieldNames) {
+            return records;
+        }
+        for(const propertiesFieldName of propertiesFieldNames){
+            const recordsWithProperties = records.filter(record => record.data[propertiesFieldName]);
+            for(const record of recordsWithProperties){
+                const dataProperties = record.data[propertiesFieldName];
+                for(const propertyData of Object.values(dataProperties)){
+                    record.data[this.getPropertyFieldName(propertyData.name, propertiesFieldName)] = propertyData;
+                }
+            }
+        }
+        return records
+    }
+
+    /**
+     * returns fields for all property defined accross all loaded records
+     * @returns {Iterable} containing definitions as fields {fieldName: [propertyValues]}
+     */
+    getPropertyDefinitions() {
+        const propertiesData = {};
+        for(const record of this.props.list.records) {
+            for(const propertiesFieldName of this.getPropertiesFieldNames()){
```

Comment:
```suggestion
            for(const propertiesFieldName of this.getPropertiesFieldNames()) {
```

(check also other occurrences)

Reply:
Mm, I just keep git prune-ing eslint out of my workspace somehow